index.css

```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: Arial, sans-serif;
+    background: #f5f5f5;
+}
+
+#todo-app {
+    max-width: 600px;
```

Comment:
We generally prefer `rem`, `em` over `px`: https://elementor.com/help/whats-the-difference-between-px-em-rem-vw-and-vh/#:~:text=EM%20is%20relative%20to%20the,parent%20size%20is%2C%20use%20REM.

-5